doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -405,6 +405,13 @@ Bluetooth Mesh
 
 The issues in this section are related to the :ref:`ug_bt_mesh` protocol.
 
```

Comment:
Please add to Bluetooth Mesh:
```
.. rst-class:: v3-0-0 v2-9-0-nRF54H20-1 v2-9-1 v2-9-0

NCSDK-31962: Advertiser buffer leak on Suspend-Resume
  Calling :c:func:`bt_mesh_adv_disable()` does not unreference advertising buffers that were previously allocated.
  Over multiple suspend-resume cycles, this can lead to buffer leaks and eventually exhaust the advertising buffer pool.
```